test/e2e-tests/support/options.go

```diff
@@ -55,6 +57,10 @@ func setOptions() (*Options, error) {
 		return options, fmt.Errorf("env variable %v not set, but is required to run tests", ClusterNamespaceEnvKey)
 	}
 
+	if options.SelfSignedCert = os.Getenv(SelfSignedCertEnvKey); options.SelfSignedCert == "" {
+		fmt.Printf("\noptional env variable %v not set, set it to use self signed certs", SelfSignedCertEnvKey)
```

Comment:
I think what Landon meant by

> Can you add a newline so that this message is separated from any follow-up output messages

was
```suggestion
		fmt.Printf("optional env variable %v not set, set it to use self signed certs\n", SelfSignedCertEnvKey)
```

Reply:
My mistake, all done